### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by hantmac**
- fix typo

### CODE REVIEW

File: website/docs/reference/warehouse-setups/databend-setup.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ meta:
   core_version: 'v1.0.0 and newer'
   cloud_support: Not Supported
   min_supported_version: 'n/a'
-  slack_channel_name: '#db-databend-cloud'
+  slack_channel_name: '#dbt-databend-cloud'
   slack_channel_link: 'https://datafusecloud.slack.com/archives/C04H167UABF'
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/databend-setup.md]: does this link exist? You should request a channel in the #channel-requests channel of our Slack

**Comment by mirnawong1** [comment]
hey @hantmac ! wanted to check in on this pr and confirm whether you've been able to get that channel created. many thanks!

**Comment by hantmac** [comment]
> hey @hantmac ! wanted to check in on this pr and confirm whether you've been able to get that channel created. many thanks!

Sorry, I want to create a new channel in dbt-labs slack but not got enough votes.

File: website/docs/reference/warehouse-setups/databend-setup.md (hunk 1 above)
Comment (hantmac) [reply website/docs/reference/warehouse-setups/databend-setup.md]: Yes, this link exists. I want to create a new channel in dbt-labs slack but not got enough votes.

Comment (mirnawong1) [reply website/docs/reference/warehouse-setups/databend-setup.md]: hey @hantmac thanks for clarifying that! the slack channel link is different to the slack channel name so if you update the slack channel name, that can clarify things for users. let me know what you think!

**Comment by mirnawong1** [comment]
hey @hantmac thanks for clarifying that! the slack channel link is different from the slack channel name so if you update the slack channel name, that can clarify things for users. or we can remove the slack line. 

let me know what you think!

**Comment by hantmac** [comment]
> hey @hantmac thanks for clarifying that! the slack channel link is different from the slack channel name so if you update the slack channel name, that can clarify things for users. or we can remove the slack line.
> 
> let me know what you think!

Ok. Thanks for your reply and I prefer to remove the slack link.

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
> > hey @hantmac thanks for clarifying that! the slack channel link is different from the slack channel name so if you update the slack channel name, that can clarify things for users. or we can remove the slack line.
> > let me know what you think!
> 
> Ok. Thanks for your reply and I prefer to remove the slack link.

thanks @hantmac for removing that! I've approved and merged this pr for you 😄 !

**Pull request merged**